src/main/webapp/app/course/manage/image-cropper-modal.component.ts

```diff
@@ -0,0 +1,35 @@
+import { Component } from '@angular/core';
+import { NgbActiveModal } from '@ng-bootstrap/ng-bootstrap';
+import { ImageCroppedEvent } from 'app/shared/image-cropper/interfaces/image-cropped-event.interface';
+
+@Component({
+    selector: 'jhi-image-cropper-modal',
+    templateUrl: './image-cropper-modal.component.html',
+})
+export class ImageCropperModalComponent {
+    courseImageUploadFile?: File;
+    croppedImage?: string;
+
+    constructor(private activeModal: NgbActiveModal) {}
+
+    /**
+     * @param event
+     */
```

Comment:
I'd say either add a description or remove the comment

Reply:
Description is added